suomen-vaylat-app/src/components/gfi/GFIPopup.jsx

```diff
@@ -517,17 +518,24 @@ export const GFIPopup = ({ handleGfiDownload }) => {
     const tablePropsInit = (data) => {
         const properties = data && data.content && data.content[0] && data.content[0].geojson && data.content[0].geojson.features && data.content[0].geojson.features[0].properties;
 
-        var hightPriorityColumns = properties?._orderHigh && JSON.parse(properties?._orderHigh);
+        var highPriorityColumns = properties?._orderHigh && JSON.parse(properties?._orderHigh);
         var lowPriorityColumns = properties?._order && JSON.parse(properties?._order);
+        var filteringColumns = properties?.filterFields && JSON.parse(properties?.filterFields);
```

Comment:
logiikkaa ei varmaan saada toimimaan näin että jokaisella featurella olisi ominaisuustietoina filterfieldsit, geoportaaliin lisätyt attribuutti kentät eivät kopioidu jokaislle featurelle, tähän pitänee keksiä joku tapa, jolla hakea erikseen per taso filteröintikentät ja tietotyypit